### PR TITLE
Address Issue 63

### PR DESCRIPTION
**Pull request by mikelodder7**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/did-spec/pull/93.html" title="Last updated on Jul 2, 2018, 5:03 PM GMT (879b73d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/did-spec/93/5e758f2...879b73d.html" title="Last updated on Jul 2, 2018, 5:03 PM GMT (879b73d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1026,6 +1026,12 @@ <h2>Public Keys</h2>
 private keys are controlled by the entity identified by the DID ("owned" public keys).
 However, a DID Document MAY also list "non-owned" public keys.
 </p>
+<p>
+Public keys that do not exist in the DID Document MUST assume the key has been revoked or is invalid.
```
Comment (msporny) [inline index.html]: Minor language nit - the way this is written, it seems as if you're saying "The public key must assume that it has been revoked/is invalid"... what about this instead:

"If a public key does not exist in a DID Document, it MUST be assumed that the key has been revoked or is invalid."

File: index.html [hunk 2]
```diff
@@ -1026,6 +1026,12 @@ <h2>Public Keys</h2>
...
+The DID Document MAY contain revoked keys. Revoked keys that remain in the DID Document MUST have its associated
```
Comment (msporny) [inline index.html]: Language nit: Plural "keys" with singular "its" - also, use e.g for examples -- "like a list" Suggest:

"A DID Document that contains a revoked key MUST also contain or refer to the revocation information for the key (e.g. a revocation list).

Comment (mikelodder7) [reply index.html]: Accepted

File: index.html (hunk 1 above)
Comment (mikelodder7) [reply index.html]: Accepted

**Comment by msporny** [comment]
LGTM, merged. Thanks @mikelodder7!

**Pull request merged**